Packaging/Packaging.wapproj

```diff
@@ -0,0 +1,116 @@
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|ARM">
+      <Configuration>Debug</Configuration>
+      <Platform>ARM</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|ARM">
+      <Configuration>Release</Configuration>
+      <Platform>ARM</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|ARM64">
+      <Configuration>Debug</Configuration>
+      <Platform>ARM64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|ARM64">
+      <Configuration>Release</Configuration>
+      <Platform>ARM64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <PropertyGroup>
+    <WapProjPath Condition="'$(WapProjPath)'==''">$(MSBuildExtensionsPath)\Microsoft\DesktopBridge\</WapProjPath>
+  </PropertyGroup>
+  <Import Project="$(WapProjPath)\Microsoft.DesktopBridge.props" />
+  <PropertyGroup>
+    <ProjectGuid>05d4c6e5-d33d-4c15-91fe-3dd2b6a5d61c</ProjectGuid>
+    <!-- ToolingVersions.props included above does not seem to affect WAPProj, so repeat them here -->
+    <TargetPlatformVersion>10.0.22621.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.19041.0</TargetPlatformMinVersion>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <AppxPackageSigningEnabled>True</AppxPackageSigningEnabled>
+    <NoWarn>$(NoWarn);NU1702</NoWarn>
+    <EntryPointProjectUniqueName>..\src\DevHome.csproj</EntryPointProjectUniqueName>
+    <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
+    <AppxPackageSigningTimestampDigestAlgorithm>SHA256</AppxPackageSigningTimestampDigestAlgorithm>
+    <AppxAutoIncrementPackageRevision>False</AppxAutoIncrementPackageRevision>
+    <GenerateTestArtifacts>True</GenerateTestArtifacts>
+    <AppxBundlePlatforms>x86|x64</AppxBundlePlatforms>
+    <HoursBetweenUpdateChecks>0</HoursBetweenUpdateChecks>
+    <BuildRing Condition="'$(BuildRing)'==''">Dev</BuildRing>
+    <AppxPackageSigningEnabled>False</AppxPackageSigningEnabled>
+  </PropertyGroup>
+  <!-- Default asset scale is 200% but DEV only has 100% scaling images available-->
+  <PropertyGroup Condition="'$(BuildRing)' == 'Dev'">
+    <UapDefaultAssetScale>100</UapDefaultAssetScale>
+  </PropertyGroup>
+  <ItemGroup>
+    <AppxManifest Include="Package.appxmanifest">
+      <SubType>Designer</SubType>
+    </AppxManifest>
+  </ItemGroup>
+  <!-- To build with the correct logo assets, only include the ones for the current build ring.
+       Place them in a common location, so references to them don't need to adjust for build ring. -->
+  <ItemGroup>
+    <Content Include="..\src\Assets\*" />
+    <Content Remove="..\src\Assets\Preview\*" />
+    <Content Remove="..\src\Assets\Canary\*" />
+    <Content Remove="..\src\Assets\Dev\*" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(BuildRing)' == 'Dev'">

```

Comment:
Can these three conditions be consolidated into one node using $(BuildRing)?  Example `<Content Include"..\src\assets\$(BuildRing)`?

Reply:
this is being deleted, it was remnants from a previous solution